ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/cco-manual-workload-identity/disconnected/cucushift-installer-rehearse-azure-ipi-cco-manual-workload-identity-disconnected-workflow.yaml

```diff
@@ -0,0 +1,15 @@
+workflow:
+  as:  cucushift-installer-rehearse-azure-ipi-cco-manual-workload-identity-disconnected
```

Comment:
Compared with the existing ci "cucushift-installer-rehearse-gcp-ipi-disconnected-private-cco-manual-workload-identity", I think we'd better to make them consistent.

Reply:
@jianping-shu  Thank you, there is a problem with the installation at present, and it will be changed together after the debugging.